sdk/core/azure-core/README.md

```diff
@@ -2,21 +2,51 @@
 
 Azure::Core (`azure-core`) provides shared primitives, abstractions, and helpers for modern Azure SDK client libraries written in the C++. These libraries follow the [Azure SDK Design Guidelines for C++][azure_sdk_cpp_development_guidelines].
 
-The library allows client libraries to expose common functionality in a consistent fashion.  Once you learn how to use these APIs in one client library, you will know how to use them in other client libraries.
+The library allows client libraries to expose common functionality in a consistent fashion. Once you learn how to use these APIs in one client library, you will know how to use them in other client libraries.
 
 ## Getting started
 
-Typically, you will not need to download `azure-core`; it will be downloaded for you as a dependency of the client libraries.  In case you want to download it explicitly (to implement your own client library, for example), you can find the source
-in here.
+Typically, you will not need to download `azure-core`; it will be downloaded for you as a dependency of the client libraries. In case you want to download it explicitly (to implement your own client library, for example), you can find the source in here, or use vcpkg to install the package `azure-core-cpp`.
 
 ## Key concepts
 
-The main shared concepts of Azure::Core include:
-- Configuring service cliesnt, e.g. configuring retries, logging, etc.. (`ClientOptions`)
-- Accessing HTTP response details (`Response`, `Response<T>`)
-- Polling long-running operations
-- Exceptions for reporting errors from service requests in a consistent fashion (`RequestFailedException`)
-- Abstractions for Azure SDK Credentials (`TokenCredential`)
+The main shared concepts of `Azure::Core` include:
+
+- HTTP pipeline and HTTP policies such as retry and logging, which are configurable via service client specific options.
+- Handling streaming data and input/output (I/O) via `BodyStream` along with its derived types.
+- Accessing HTTP response details from the method return types, via `Response<T>`.
```

Comment:
`from the method return types` .. which method?
maybe
`Wrapped Response<T> for the returned model of an sdk client operation and its http raw response`